relay-server/src/actors/processor.rs

```diff
@@ -1079,7 +1079,7 @@ impl EnvelopeProcessorService {
     }
 
     /// Remove replays if the feature flag is not enabled
-    fn process_replays(&self, state: &mut ProcessEnvelopeState) {
+    fn process_replays(&self, state: &mut ProcessEnvelopeState) -> Result<(), ProcessingError> {
```

Comment:
I think we can revert the signature change here now, right?

```suggestion
    fn process_replays(&self, state: &mut ProcessEnvelopeState) {
```